test/unit/csv_exporter_test.rb

```diff
@@ -26,4 +26,15 @@ class CsvExporterTest < ActiveSupport::TestCase
       result.next
     end
   end
+
+  test 'accepts custom column headers' do
+    result = CsvExporter.export(Host::Managed, [:id], ['My Lovely Header'])
```

Comment:
would the following be a better interface?
```ruby
CsvExporter.export(Host::Managed, [:id => ['My Lovely Header'])
```
or something along those lines? is there any other attribute besides rename?

if nothing else, I would prefer ```[[header, title],[header,title]]```..